runtime/vm/threadhelp.cpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 1998, 2020 IBM Corp. and others
+ * Copyright (c) 1998, 2021 IBM Corp. and others
```

Comment:
Why is this file being changed? Wouldn't this be better in one of the JNI support files?

Reply:
I was thinking it tied to J9VMThread at first... would `jnimisc.cpp` be a better home for these functions?

Reply:
I think so, yes.

Reply:
I migrated over the implementations to `jnimisc.cpp`, and the function prototypes to `vm_internal.h`.   Considered making them VM internal functions, as the JIT DirectToJNI dispatch may need to query the matching 31-bit jniEnv if not resolved at JNI call.  Although, given the complexity of CEL4RO31 dispatch and handling, will need to consider which scenarios we want / can effectively build JIT dispatch support.  I propose we move only expose these functions to VM internal functions if needed.

https://github.com/eclipse/openj9/pull/12074/commits/e2c7aa733250dfc7af3d882992feb4d5195e5fda